distribution/src/config/jvm.options

```diff
@@ -45,6 +45,14 @@
 # 10-:-XX:+UseG1GC
 # 10-:-XX:InitiatingHeapOccupancyPercent=75
 
+## DNS cache policy
+# cache ttl for positive DNS lookups noting that this overrides the JDK security
+# property networkaddress.cache.ttl; set to -1 to cache forever
+-Des.networkaddress.cache.ttl=60
```

Comment:
I wonder if we should default these in the code and only override the setting if we detect it has not been overridden in the `security.policy` file.

I’m not sure if we can do that check appropriately before we init the manager though.

Reply:
We would not be able to distinguish between someone that set `-1` because it is the default, and someone that set `-1` because it is the value that they want. Therefore I opt to say: this is how Elasticsearch does it, if you don't want it and want it controlled at the system level, remove these properties.

Reply:
That's a fair point. It's frustrating that `Security.getProperty` doesn't take a default to allow you to detect the difference. It'll just need to be called out in the release notes that anyone that has set this really needs to change this if they disagree with the "new" setting.